CMakeLists.txt

```diff
@@ -68,6 +68,13 @@ list(APPEND COMPUTECPP_USER_FLAGS
   -no-serial-memop
 )
 
+if (MSVC)
+  list(APPEND COMPUTECPP_USER_FLAGS
+    -D_ALLOW_COMPILER_AND_STL_VERSION_MISMATCH=1
```

Comment:
I believe the ComputeCpp SDK already contains a proper fix for this in `FindComputeCpp`, so a better fix would be to update the SDK submodule. However this is probably fine for now.

Reply:
You are correct. I think I will update the submodule as well

Reply:
I've updated the computecpp-sdk submodule and I changed the way to set the c++ standard using cmake `set(CMAKE_CXX_STANDARD 14)` when on Windows